LICENSE-binary

```diff
@@ -521,6 +521,11 @@ junit:junit:4.13.2
 org.jacoco:org.jacoco.agent:0.8.5
 
 
+Eclipse Public License 2.0
+--------------------------
+
+com.github.jnr:jnr-posix:3.1.19
```

Comment:
it also pulls transitive jars.

Honestly, it's a bit heavy to pull bunches of jars to just work around the access of `sun.misc.Signal` API. Could it be addressed by using reflection?

Reply:
@pan3793 This change is likely based on related modifications in Ozone. If reflection is used, could you provide some specific implementation ideas?

Ozone: 
JIRA: HDDS-11078. Remove usage of sun.misc.Signal
https://github.com/apache/ozone/pull/7006
https://github.com/apache/ozone/pull/6876

@myandpr Is there any additional information?

Reply:
Exactly, and the problem is that sun.misc package is no longer supported after JDK 9，even  removed in a future release( related information: [https://openjdk.org/jeps/260](https://openjdk.org/jeps/260)).

Reply:
Exactly, and the problem is that sun.misc package is no longer supported after JDK 9，even  removed in a future release( related information: [https://openjdk.org/jeps/260](https://openjdk.org/jeps/260)). @slfan1989 @pan3793